src/components/LoginForm/index.js

```diff
@@ -32,30 +49,48 @@ const LoginForm = ({ onSubmit }) => {
 
   const inputProps = useTextInputProps(handleValueChange, handleBlur, values);
 
+  const buttonTitle = useMemo(
```

Comment:
What do you think about creating a custom button component that handles all this logic, and via props you can control the loading. If not you can imagine yourself writing this code on every form